ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -784,3 +784,58 @@ def test_numpy_gamma(
         scale=scale,
         size=size,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.random.triangular",
+    input_dtypes=helpers.get_dtypes("float"),
+    left=st.floats(
+        allow_nan=False,
+        allow_infinity=False,
+        width=32,
+        min_value=0,
+        max_value=10,
+    ),
+    mode=st.floats(
+        allow_nan=False,
+        allow_infinity=False,
+        width=32,
+        min_value=10,
+        max_value=100,
+        exclude_min=True,
+    ),
+    right=st.floats(
+        allow_nan=False,
+        allow_infinity=False,
+        width=32,
+        min_value=100,
+        max_value=1000,
+        exclude_min=True,
+    ),
+    size=helpers.get_shape(allow_none=False),
```

Comment:
May be `allow_none` should be set to True given that the default values of size is None

Reply:
updated it , thanks